monkey/tests/unit_tests/infection_monkey/telemetry/messengers/test_credentials_intercepting_telemetry_messenger.py

```diff
@@ -0,0 +1,68 @@
+from unittest.mock import MagicMock
+
+from infection_monkey.credential_collectors import Password, SSHKeypair, Username
+from infection_monkey.i_puppet import Credentials
+from infection_monkey.telemetry.base_telem import BaseTelem
+from infection_monkey.telemetry.credentials_telem import CredentialsTelem
+from infection_monkey.telemetry.messengers.credentials_intercepting_telemetry_messenger import (
+    CredentialsInterceptingTelemetryMessenger,
+)
+
+TELEM_CREDENTIALS = [
+    Credentials(
+        [Username("user1"), Username("user3")],
+        [
+            Password("abcdefg"),
+            Password("root"),
+            SSHKeypair(public_key="some_public_key", private_key="some_private_key"),
+        ],
+    )
+]
+
+
+class TestTelem(BaseTelem):
+    telem_category = None
+    __test__ = False
+
+    def __init__(self):
+        pass
+
+    def get_data(self):
+        return {}
```

Comment:
Does this do anything differently than the `TestTelem` in `test_expliot_intercepting_telemetry_messenger`? If not, let's put it some place common. Maybe `tests/unit_tests/infection_monkey/telemetry/messengers/test_telem.py` Or as a fixture in `conftest.py`:

```python
@pytest.fixture(scope="package")
def TestTelem():
    class InnerTestTelem(BaseTelem):
        ...

    return InnerTestTelem
```